cmd/event_receiver/main.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"github.com/knative/eventing/pkg/event"
+	"log"
+	"net/http"
+	"time"
+)
+
+type Heartbeat struct {
```

Comment:
so... not sure, but could this `struct` be moved to a common place? it's now around twice ... sure it's demo ... but not sure if that could be in `pkg/hearbeat`

Reply:
I don't really want to because they are a demo and I was trying to keep them independent.